modules/jupyter-widget/src/widget.js

```diff
@@ -103,9 +119,11 @@ export function initDeck({mapboxApiKey, container, jsonInput}) {
           latitude: 0,
           longitude: 0,
           zoom: 1,
+          onClick: handleClick,
           container,
           onLoad: () => updateDeck(jsonInput, {jsonConverter, deckConfig})
         });
+        context.jsonDeck = {jsonConverter, deckConfig};
```

Comment:
This function modifying `context`, instead of returning these values and having the caller add them to `context` raises flags. 

Modification of arguments is a subtle source of bugs and should be avoided whenever possible unless justified by a strong reason or a specific design choice. I see evidence of neither here.